### ISSUE TITLE
Plans to add MessagePack?

### ISSUE THREAD

**Issue by CheKhenKho** [jamiewest/signalr_core#35]
Would be nice to know if you plan to add MessagePack support in the future.

**Comment by jamiewest** [comment]
I would like to, just need to invest some time. Do you have a project that would use it?

**Comment by CheKhenKho** [comment]
yeah, football livescores. Need to send live events, stats, lineups, etc. to a bunch of clients every minute of every game. Right now I serialize to messagepack manually, then compress with brotli, and convert to base64. msgpack_dart package works quite well, if you are thinking of integrating an existing solution.
